mod/follow.php

```diff
@@ -175,12 +175,12 @@ function follow_post(&$a) {
 			notice($result['message']);
 		goaway($return_url);
 	} elseif ($result['cid'])
-		goaway($a->get_baseurl().'/contacts/'.$result['cid']);
+		goaway(App::get_baseurl().'/contacts/'.$result['cid']);
 
 	info( t('Contact added').EOL);
 
 	if(strstr($return_url,'contacts'))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊